### PR TITLE
Extracted content-list from my-objects

### PR DESCRIPTION
**Pull request by ktonon**

### CODE REVIEW

File: src/components/content-list.js [hunk 1]
```diff
@@ -0,0 +1,139 @@
...
+		this.loadPage();
+	}
+
+	async loadPage() {
+		this.contentItems = await this.apiClient.listContent();
+	}
+
+	render() {
+		debugger;
+		return html`
```
Comment (ktonon) [inline src/components/content-list.js]: I'm trying to extract `<content-list>` from `<my-objects>`. I can see that both the constructor and `connectedCallback()` get invoked, but this `render()` is never hit. No idea what's going on...

Comment (ktonon) [reply src/components/content-list.js]: No console errors

Comment (qwangd2l) [reply src/components/content-list.js]: I believe the issue is that you're extending PageViewElement
and that has logic on what's active to trigger an update or not.
I'd just put LitElement here

Comment (ktonon) [reply src/components/content-list.js]: Yes, that was it. Thank you!

**Pull request merged**